orion-server/src/main/java/com/pinterest/orion/core/kafka/KafkaCluster.java

```diff
@@ -122,6 +124,12 @@ public int getClusterDefaultReplicationFactor() {
   @Override
   public void bootstrapClusterInfo(Map<String, Object> config) {
     props = new Properties();
```

Comment:
I considered just updating the properties here, but all the other Cluster's in this repo seemed to pretty consistently be using attributes to move this kind of info around.

Reply:
Worth noting that I don't see anywhere where this field is updated.